addons/mail/static/src/core/common/message_model.js

```diff
@@ -32,6 +33,22 @@ export class Message extends Record {
     attachment_ids = Record.many("ir.attachment", { inverse: "message" });
     author = Record.one("Persona");
     body = Record.attr("", { html: true });
+    richBody = Record.attr("", {
+        compute() {
+            if (!this.store.emojiLoader.loaded || !this.body) {
```

Comment:
Shouldn't we trigger a load emoji when displaying a message with an emoji inside it?

So if loaded is accessed but not yet present, we trigger it?

I guess it should be a function then (or a smart getter).

Reply:
We can trigger it and the compute will be re-evaluated when load is done, no need to change to a getter or a function imo.